test/routes/stats.test.js

```diff
@@ -1,10 +1,13 @@
+const { init } = require('@forestadmin/context');
 const Stats = require('../../src/routes/stats');
 
 describe('routes > stats', () => {
   describe('.get', () => {
     it('should return a value for objective chart', async () => {
       expect.assertions(1);
 
+      init((ctx) => ctx);
```

Comment:
Could please explain why we need to setup context now ? Compared to none before ?

Reply:
We do since `context.inject` was always defined in the previous implementation, returning `{}` when no context was injected.

The context library prevent using `inject` before `init` (throwing an error), so tis lien is definitely required :)

Reply:
👍 🙏